lib/cryptoexchange/exchanges/bigone/services/market.rb

```diff
@@ -0,0 +1,36 @@
+module Cryptoexchange::Exchanges
+  module Bigone
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
```

Comment:
Since it is possible to get all the tickers using `https://api.big.one/markets`, this would be preferred vs making multiple individual API calls. Consider implementing `supports_individual_ticker_query?` as false and the corresponding adapt for a single API call.